rln-js/index.html

```diff
@@ -56,17 +56,24 @@ <h4>You can either generate new credentials:</h4>
       </div>
       <div class="w50">
         <h4>Or import existing ones:</h4>
-        <label for="membership-id"
-          >Membership ID (your index in the RLN smart contract):</label
-        >
-        <input id="membership-id" name="membership-id" type="text" />
-        <label for="id-key">RLN Identity Key (hex string):</label>
-        <input id="id-key" name="id-key" type="text" />
-        <label for="commitment-key">RLN Commitment Key (hex string):</label>
-        <input id="commitment-key" name="commitment-key" type="text" />
-        <button disabled id="import-button" type="button">
-          Import RLN Credentials
-        </button>
+        <div>
+          <label for="membership-id"
+            >Membership ID (your index in the RLN smart contract):</label
+          >
+          <input id="membership-id" name="membership-id" type="text" />
+          <label for="id-key">RLN Identity Key (hex string):</label>
+          <input id="id-key" name="id-key" type="text" />
+          <label for="commitment-key">RLN Commitment Key (hex string):</label>
+          <input id="commitment-key" name="commitment-key" type="text" />
+          <button disabled id="import-manually-button" type="button">
+            Import RLN Credentials
+          </button>
+        </div>
+        <div id="import-from-wallet">
+          <button disabled id="import-from-wallet-button" type="button">
+            Import RLN Credentials from Wallet
```

Comment:
I think we should have only one button that generate the credentials and tries to retrieve from the blockchain.

Which means the user always click on "Generate Credentials from Wallet".

Then, depending on whether it found previous credentials, then it either offer to insert (no credential found) or ready to go (credentials found).

If we want, we may add a Status message that can help guide the user if it gets confusing.

Reply:
Makes sense; 
https://www.loom.com/share/408c1cc6ec854dde8cfe38398403538f

Reply:
Looks good.

It says "no proof attached" in your messages. Is it because of the new zerokit API? **this is blocking**.

Reply:
interesting. 
unsure why that is happening: confirmed that this is also a case on the current `master`/https://examples.waku.org/rln-js/ @fryorcraken 

cc @richard-ramos @rymnc

Reply:
update: upon debugging this with @rymnc running a local nwaku node with rln enabled, the proof is attached and works as expected.
it's safe to conclude that the fleets currently are *not* running RLN causing this example to break.
cc @fryorcraken